src/components/accordion/accordion.component.tsx

```diff
@@ -1,46 +1,43 @@
-import React, { useState, useRef, useEffect, useCallback } from "react";
+import React, { useRef, useState } from "react";
 import { SpaceProps } from "styled-system";
-import tagComponent, { TagProps } from "../../__internal__/utils/helpers/tags";
 
-import useResizeObserver from "../../hooks/__internal__/useResizeObserver";
+import Box from "../box";
+import Typography from "../typography";
+import useMediaQuery from "../../hooks/useMediaQuery";
 import createGuid from "../../__internal__/utils/helpers/guid";
-import Events from "../../__internal__/utils/helpers/events";
+import tagComponent, { TagProps } from "../../__internal__/utils/helpers/tags";
+import ValidationIcon from "../../__internal__/validations";
+
 import {
-  StyledAccordionContainer,
-  StyledAccordionHeadingsContainer,
-  StyledAccordionTitleContainer,
-  StyledAccordionTitle,
-  StyledAccordionSubTitle,
-  StyledAccordionIcon,
-  StyledAccordionContentContainer,
-  StyledAccordionContent,
-  StyledAccordionContainerProps,
+  StyledContent,
+  StyledDetails,
+  StyledSummary,
+  StyledWrapper,
+  StyledSummaryTitleWrapper,
+  StyledSummaryTitle,
+  StyledIcon,
 } from "./accordion.style";
-import ValidationIcon from "../../__internal__/validations";
 
-export interface AccordionProps
-  extends StyledAccordionContainerProps,
-    SpaceProps,
-    TagProps {
-  /** Content of the Accordion component */
+export interface AccordionProps extends SpaceProps, TagProps {
```

Comment:
comment: looks like some props have been removed (`defaultExpanded`) so this needs to be a breaking change commit